src/fu-engine.c

```diff
@@ -507,13 +508,27 @@ fu_engine_modify_device (FuEngine *self,
 	return FALSE;
 }
 
+static const gchar *
+_g_checksum_type_to_string (GChecksumType checksum_type)
+{
+	if (checksum_type == G_CHECKSUM_MD5)
+		return "md5";
```

Comment:
New library; maybe it's also time to drop support for insecure checksums?

Reply:
Ohh, I'm lobbying this function to be included into glib -- hopefully we can drop it soon.